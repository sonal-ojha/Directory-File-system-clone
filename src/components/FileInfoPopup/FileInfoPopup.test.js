import React from 'react';
import FileInfo from './FileInfoPopup.component';
import renderer from 'react-test-renderer';

it('File Info Popup', () => {
    const getInfoItem = {
        name: 'f1',
        size: '12kb',
        creator: 'self',
        date: '12/12/2019',
    };
    const handleInfoClose = jest.fn();
    const tree = renderer.create(<FileInfo closeInfo={handleInfoClose} item={getInfoItem}/>);
    expect(tree).toMatchSnapshot();
})