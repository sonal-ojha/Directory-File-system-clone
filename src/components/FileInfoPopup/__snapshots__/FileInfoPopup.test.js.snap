exports[`test File Info Popup 1`] = `
<div
  className="backgroundBlur">
  <div
    className="info-modal">
    <div
      className="displayFlex textAlignCenter fileHeader">
      <div
        className="f-18"
        style={
          Object {
            "width": "280px",
          }
        }>
        File Info
      </div>
      <button
        className="btn pointer"
        onClick={[Function]}>
        <i
          aria-hidden="true"
          className="fa fa-times fa-2x" />
      </button>
    </div>
    <div
      className="displayFlex textAlignCenter iconFileType">
      <i
        aria-hidden="true"
        className="fa fa-folder fa-5x" />
    </div>
    <div
      className="m-10">
      <div
        className="infoListItem">
        Name : 
        f1
      </div>
      <div
        className="infoListItem">
        Size : 
        12kb
      </div>
      <div
        className="infoListItem">
        Creator Name : 
        self
      </div>
      <div
        className="infoListItem">
        Created At : 
        12/12/2019
      </div>
    </div>
  </div>
</div>
`;
